src/certificates/controllers/certificates.controller.ts

```diff
@@ -99,8 +112,8 @@ export class CertificatesController {
 
   @Get('download/:id')
   @UseGuards(JwtAuthGuard)
-  download(@Param('id') id: string) {
-    return this.certificatesService.getFile(+id);
+  async download(@Param('id', ParseIntPipe) id: number) {
+    return this.certificatesService.getFile(id);
```

Comment:
Should this API call be here or in the certificate-files API?

Reply:
Hmm, really good point. I'm gonna transfer it to the certificates-files controller since it whole point is to manage files! Nice catch.